src/components/edit/Oaks.jsx

```diff
@@ -4,10 +4,14 @@ import Select from 'react-select';
 import useSWRMutation from 'swr/mutation';
 import { useAuth0 } from "@auth0/auth0-react";
 import { getOak, addOrUpdateOak } from '../../services/oaks';
-import { TextInput, TextArea } from '../shared/FormInputs';
+import { RadioGroup, TextInput, TextArea } from '../shared/FormInputs';
```

Comment:
it seems like the changes in this file are independent and should be a separate pull request